src/windowsdesktop/src/sfx/Microsoft.WindowsDesktop.App.Runtime.sfxproj

```diff
@@ -10,4 +10,95 @@
+    <FrameworkListFileClass Include="WindowsFormsIntegration.dll"  Profile="WindowsForms;WPF"/>
+    <FrameworkListFileClass Include="System.Xaml.resources.dll" Profile="WindowsForms;WPF" />
+    <FrameworkListFileClass Include="UIAutomationClient.resources.dll" Profile="WindowsForms;WPF" />
+    <FrameworkListFileClass Include="UIAutomationClientSideProviders.resources.dll" Profile="WindowsForms;WPF" />
+    <FrameworkListFileClass Include="UIAutomationProvider.resources.dll" Profile="WindowsForms;WPF" />
+    <FrameworkListFileClass Include="UIAutomationTypes.resources.dll" Profile="WindowsForms;WPF" />
+    <FrameworkListFileClass Include="WindowsBase.resources.dll" Profile="WindowsForms;WPF" />
+    <FrameworkListFileClass Include="WindowsFormsIntegration.resources.dll" Profile="WindowsForms;WPF" />
+    <FrameworkListFileClass Include="PenImc_cor3.dll" Profile="WindowsForms;WPF" />
+    <FrameworkListFileClass Include="PresentationNative_cor3.dll" Profile="WindowsForms;WPF" />
+    <FrameworkListFileClass Include="vcruntime140_cor3.dll" Profile="WindowsForms;WPF" />
+    <FrameworkListFileClass Include="wpfgfx_cor3.dll" Profile="WindowsForms;WPF" />
+    <FrameworkListFileClass Condition="'$(PlatformTarget)' != 'ARM64'" Include="D3DCompiler_47_cor3.dll" Profile="WindowsForms;WPF" />
+  </ItemGroup>
+
+  <!-- Windows Forms specific references -->
+  <ItemGroup>
+    <FrameworkListFileClass Include="Microsoft.VisualBasic.Forms.dll" Profile="WindowsForms" />
+    <FrameworkListFileClass Include="Microsoft.VisualBasic.dll" Profile="WindowsForms" />
+    <FrameworkListFileClass Include="System.Design.dll" Profile="WindowsForms" />
+    <FrameworkListFileClass Include="System.Drawing.Common.dll" Profile="WindowsForms" />
+    <FrameworkListFileClass Include="System.Drawing.Design.dll" Profile="WindowsForms" />
+    <FrameworkListFileClass Include="System.Drawing.dll" Profile="WindowsForms" />
+    <FrameworkListFileClass Include="System.Windows.Forms.Design.Editors.dll" Profile="WindowsForms" />
+    <FrameworkListFileClass Include="System.Windows.Forms.Design.dll" Profile="WindowsForms" />
+    <FrameworkListFileClass Include="System.Windows.Forms.Primitives.dll" Profile="WindowsForms" />
+    <FrameworkListFileClass Include="System.Windows.Forms.dll" Profile="WindowsForms" />
+    <FrameworkListFileClass Include="Microsoft.VisualBasic.Forms.resources.dll" Profile="WindowsForms" />
+    <FrameworkListFileClass Include="System.Windows.Forms.Design.resources.dll" Profile="WindowsForms" />
+    <FrameworkListFileClass Include="System.Windows.Forms.Primitives.resources.dll" Profile="WindowsForms" />
+    <FrameworkListFileClass Include="System.Windows.Forms.resources.dll" Profile="WindowsForms" />
+    <FrameworkListFileClass Include="System.Windows.Input.Manipulations.resources.dll" Profile="WindowsForms" />
+    <FrameworkListFileClass Include="System.Private.Windows.Core.dll"/>
+  </ItemGroup>
```

Comment:
This is an incorrect way of doing this. The Windows Forms items must be imported from System.Windows.Forms.FileClassification.props. If the content of the props file is incorrect then it must be updated: https://github.com/dotnet/winforms/tree/main/pkg/Microsoft.Private.Winforms#sdkdotnet-windowsdesktop-folder

/cc: @lonitra